src/credentials/awsCredentialsStatusBarItem.ts

```diff
@@ -29,25 +29,31 @@ export async function initializeAwsCredentialsStatusBarItem(
         'The current credentials used by the {0} Toolkit.\n\nClick this status bar item to use different credentials.',
         getIdeProperties().company
     )
+
     statusBarItem.show()
     updateCredentialsStatusBarItem(statusBarItem)
 
     context.subscriptions.push(statusBarItem)
 
     context.subscriptions.push(
-        awsContext.onDidChangeContext(async (awsContextChangedEvent: ContextChangeEventsArgs) => {
-            updateCredentialsStatusBarItem(statusBarItem, awsContextChangedEvent.profileName)
+        awsContext.onDidChangeContext(async (ev: ContextChangeEventsArgs) => {
+            updateCredentialsStatusBarItem(statusBarItem, ev.profileName, ev.developerMode)
         })
     )
 }
 
 // Resolves when the status bar reaches its final state
 export async function updateCredentialsStatusBarItem(
     statusBarItem: vscode.StatusBarItem,
-    credentialsId?: string
+    credentialsId?: string,
+    developerMode?: boolean
 ): Promise<void> {
     clearTimeout(timeoutID)
 
+    if (developerMode) {
+        ;(statusBarItem as any).backgroundColor = new vscode.ThemeColor('statusBarItem.errorBackground')
+    }
```

Comment:
I left that out because it's quite an edge-case: it would mean that the developer removed any (all) dev settings from their settings.json, while the Toolkit is running.

Reply:
added